bin/vault/console/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const genericConsole = require('bin/generic/console');
+
+module.exports = resource => genericConsole(resource,
+    (args, vault) => `https://panel.hyperone.com/console/vault/${vault._id}?tenant=${args.profile.project._id}&scope=${process.env.SCOPE_NAME}`
```

Comment:
Use project query parameter instead of tenant.

Reply:
This will corrupt the functionality of the application if you do not make changes on your side.

Reply:
Fixed.

Reply:
fixed on my side

Reply:
Cool! Ready to merge!